Assignment1/Assignment1_QNo5/SolAssignmentNo1_QNo5.rb

```diff
@@ -0,0 +1,15 @@
+# => Find out how to convert the string 'The Ruby Book' to 'THE RUBY BOOK' ?
+# => author : Hrisikesh Singha Chowdhury
+
+# => Fn to implement buisness logic
+def sent_to_upcase
+  puts "Enter the sentence"
+  sent = gets.chomp
+  # => Converting into upper case
+  puts "The required sentence with each character in upper case is #{sent.upcase}"
+end 
+
+# => self denotes main object which is an Instance of class Object
+self.sent_to_upcase
```

Comment:
@HrisikeshSinghaChowdhury Instead of calling with self please use a custom class to define a method and use that method using that class object